salt/states/blockdev.py

```diff
@@ -150,7 +157,7 @@ def formatted(name, fs_type='ext4', **kwargs):
         ret['result'] = None
         return ret
 
-    __salt__['blockdev.format'](name, fs_type, **kwargs)
+    __salt__['blockdev.format'](name, fs_type, force,  **kwargs)
```

Comment:
@abednarik, one more thing related to the API comment above, this should be `force=force`.